tools/cli/internal/cli/split/split.go

```diff
@@ -72,6 +76,20 @@ func (o *Opts) Run() error {
 	return nil
 }
 
+func (o *Opts) skipVersion(version string) bool {
```

Comment:
we should expect future version to show up for dev, the hidden-env logic should not generate it if the target is prod and generate if target is dev